forms-flow-web/src/constants/userContants.js

```diff
@@ -1,3 +1,4 @@
 //TODO make it dynamic from env
 /* istanbul ignore file */
-export const REVIEWER_GROUP = "formsflow/formsflow-reviewer";
+import {MULTITENANCY_ENABLED} from "../constants/constants";
+export const REVIEWER_GROUP = MULTITENANCY_ENABLED ? "formsflow-reviewer" : "formsflow/formsflow-reviewer";
```

Comment:
Please change MULTITENANCY_ENABLED to KEYCLOAK_ENABLE_CLIENT_AUTH, as we can have client based auth too.